<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Information Archive</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Apps-utilities-system-monitor-icon.png</normaloff>
    <normalon>Apps-utilities-system-monitor-icon.png</normalon>
    <disabledoff>Apps-utilities-system-monitor-icon.png</disabledoff>
    <disabledon>Apps-utilities-system-monitor-icon.png</disabledon>
    <activeoff>Apps-utilities-system-monitor-icon.png</activeoff>
    <activeon>Apps-utilities-system-monitor-icon.png</activeon>
    <selectedoff>Apps-utilities-system-monitor-icon.png</selectedoff>
    <selectedon>Apps-utilities-system-monitor-icon.png</selectedon>Apps-utilities-system-monitor-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>833</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="PlotWidget" name="cpuView"/>
   </item>
   <item>
    <widget class="PlotWidget" name="memoryView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
